committee-steering/governance/sig-governance.md

```diff
@@ -149,6 +148,13 @@ Issues impacting multiple subprojects in the SIG should be resolved by either:
 - Option 1: SIG Technical Leads
 - Option 2: Federation of Subproject Owners
 
+### SIG Retirement
+
+- In the event that the SIG is unable to regularly establish consistent quorum
+  or otherwise fulfill its Organizational Management responsibilities
+  - after 3 or more months it *SHOULD* be retired
```

Comment:
What does retired mean?  Can they reform?   
Also, how would you pass on ownership, b/c by definition SIGs own code.

Reply:
What needs to be spelled out here? I'm trying to avoid details because I think the sig contributor experience community management subproject can own the logistics.  I would say retired is: owns no more code, and has been moved to [community/archive](https://github.com/kubernetes/community/tree/master/archive).  Can it be revived? If there are people willing to step up, sounds great.

The specific case that is motivating this has no code, so I'm wary of setting precedent for that without having a concrete example.

Reply:
Is this to rationalize sigs that have not submitted a charter, or just don't meet?  
If so, then let's do that on steering and possibly add some clauses and more details here.

Reply:
I can't answer yes to an either/or question?  I feel like I was pretty clear what the purpose of this was in the PR description.  I agree there are refinements to be made for more cases, but I don't feel a strong need to do that right now.

Reply:
Could you just open up another issue that cross-references this one spelling out that we need to clarify details on retirement and policy and we can get this one done.

Reply:
I think #2029 is that issue, so I retitled this, and added a comment there